src/rpc/mempool.cpp

```diff
@@ -867,6 +875,19 @@ static RPCHelpMan submitpackage()
                                    "Array must contain between 1 and " + ToString(MAX_PACKAGE_COUNT) + " transactions.");
             }
 
+            // Fee check needs to be run with chainstate and package context
+            const CFeeRate max_raw_tx_fee_rate = request.params[1].isNull() ?
+                                                     DEFAULT_MAX_RAW_TX_FEE_RATE :
+                                                     CFeeRate(AmountFromValue(request.params[1]));
+            std::optional<CFeeRate> max_sane_feerate{max_raw_tx_fee_rate};
+            // 0-value is special; it's mapped to no sanity check
+            if (max_raw_tx_fee_rate == CFeeRate(0)) {
+                max_sane_feerate = std::nullopt;
+            }
+
+            // Burn sanity check is run with no context
+            const CAmount max_burn_amount = request.params[2].isNull() ? 0 : AmountFromValue(request.params[2]);
+
```

Comment:
We did not document this anywhere but the default behavior  if `maxburnamount` is not passed it we are rejecting the package if there is any transactions with unspendable output greater than 0.
Maybe indicate it in the `RPCHelpMan` of `maxburnamount`?

---

Its also not indicated for `sendrawtransaction`.

Reply:
added a mention of the default being 0